### PR TITLE
bump: bump up Notation CLI to v1.1.0

### PR DESCRIPTION
**Pull request by Two-Hearts**

### CODE REVIEW

File: .github/workflows/e2e-test-setup.yml
Comment (shizhMSFT) [inline .github/workflows/e2e-test-setup.yml:1]: It is not necessary to change this e2e for each new version entry of `notation`. Basically, line 28-29 can cover the default version change.

**Review by shizhMSFT — APPROVED** [review]
LGTM with suggestions.

We need to publish a patch version for this GitHub action.

File: .github/workflows/e2e-test-setup.yml
Comment (Two-Hearts) [reply .github/workflows/e2e-test-setup.yml:1]: Will keep them for this release. Will keep in mind for our next minor release.

**Review by shizhMSFT — APPROVED** [review]
LGTM

**Comment by shizhMSFT** [comment]
@FeynmanZhou As mentioned in the community discussion, we follow SemVer2 for versioning so as other repositories. That is, since there is no new features for `notation-action`, the new version should be `v1.0.1`.

**Pull request merged**